### PR TITLE
Documentation for new parameters in logstash-output-opensearch 2.0.0

### PR DESCRIPTION
**Pull request by deepdatta**
Add documentation for 3 new parameters added in logstash-output-opensearch 2.0.0

Signed-off-by: Deep Datta <deedatta@amazon.com>

### Description
Add documentation for 3 new parameters added in logstash-output-opensearch 2.0.0
https://github.com/opensearch-project/logstash-output-opensearch/releases/tag/2.0.0

### Issues Resolved
https://github.com/opensearch-project/logstash-output-opensearch/pull/174

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _clients/logstash/ship-to-opensearch.md
Comment (Naarcha-AWS) [inline _clients/logstash/ship-to-opensearch.md:148]:
```suggestion
- service_name (string, default => "es") - Defines the service name to be used for `aws_iam` authentication.
```

File: _clients/logstash/ship-to-opensearch.md
Comment (Naarcha-AWS) [inline _clients/logstash/ship-to-opensearch.md:147]:
```suggestion
- template_name (string, default => "logstash") - Defines how the template is named inside Opensearch
```

**Review by Naarcha-AWS — APPROVED** [review]

File: _clients/logstash/ship-to-opensearch.md
Comment (Naarcha-AWS) [inline _clients/logstash/ship-to-opensearch.md:146]: Remove "if you so desire".
```suggestion
- template (path) - You can set the path to your own template here. If not template is specified, the included template will be used.
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]

File: _clients/logstash/ship-to-opensearch.md
Comment (Naarcha-AWS) [inline _clients/logstash/ship-to-opensearch.md:146]:
```suggestion
- template (path) - You can set the path to your own template here. If no template is specified, Logstash uses the default template.
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]

**Comment by Naarcha-AWS** [comment]
@deepdatta: Your PR require DCO sign-off before we can merge. For instructions on how to amend your commits, try an [interactive rebase](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md#fix-option-3-interactive-rebase).

**Comment by deepdatta** [comment]
Made the change and added Signoff

File: _clients/logstash/ship-to-opensearch.md
Comment (Naarcha-AWS) [inline _clients/logstash/ship-to-opensearch.md:149]: Fix spelling for "composable". 
```suggestion
- legacy_template (boolean, default => true) - Selects the OpenSearch template API. When `true`, uses legacy templates via the _template API. When `false`, uses composable templates via the _index_template API.
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]
One last change here.

**Review by Naarcha-AWS — APPROVED** [review]

**Review by lizsnyder — APPROVED** [review]

**Pull request merged**
